Application/DefaultSettings.json

```diff
@@ -1115,6 +1115,59 @@
       "helicopter": "Attack Helicopter",
       "player": "",
       "attach": ""
-    }
+    },
+    "T4": {	
+	  "torso_upper": "Upper Torso",
+	  "torso_lower": "Lower Torso",
+	  "right_leg_upper": "Upper Right Leg",
+	  "right_leg_lower": "Lower Right Leg",
+	  "right_hand": "Right Hand",
+	  "right_foot": "Right Foot",
+	  "right_arm_upper": "Upper Right Arm",
+	  "right_arm_lower": "Lower Right Arm",
+	  "left_leg_upper": "Upper Left Leg",
+	  "left_leg_lower": "Lower Left Leg",
+	  "left_hand": "Left Hand",
+	  "left_foot": "Left Foot",
+	  "left_arm_upper": "Upper Left Arm",
+	  "left_arm_lower": "Lower Left Arm",
+	  "gl": "Rifle Grenade",
+	  "bigammo": "Round Drum",
+	  "scoped": "Sniper Scope",
+	  "telescopic": "Telescopic Sight",
+	  "aperture": "Aperture Sight",
+	  "flash": "Flash Hider",
+	  "silenced": "Silencer",
+	  "tabun_gas": "Tabun Gas",
```

Comment:
So the way the parser works is it always splits the weapons with _ into two parts
tabun and gas. 
It also automatically converts it to title case so this one is not needed